apps/site/src/views/Home/sections/Intro/Intro.module.scss

```diff
@@ -12,6 +12,19 @@
 
 	h2 {
 		margin-bottom: 70px;
+		@include bootstrap.media-breakpoint-down(sm) {
+			margin-top: 20px;
+			margin-bottom: 50px;
+		}
+	}
+
+	@include bootstrap.media-breakpoint-down(sm) {
+		background-image: url("~@/assets/images/index-card-mobile.svg");
+		background-position: top;
+		background-size: cover;
+		box-shadow:
+			4px 4px 5px 4px rgba(0, 0, 0, 0.2),
+			-1px -1px 8px 4px rgba(0, 0, 0, 0.2);
 	}
```

Comment:
Nitpick: would appreciate a mobile-first approach but looks fine otherwise.
Question: could the `background-position: top` and `background-size: cover` also be used for the large view or would it look off?